lib/fake_async.dart

```diff
@@ -281,7 +286,8 @@ class FakeTimer implements Timer {
   late Duration _nextCall;
 
   /// The current stack trace when this timer was created.
```

Comment:
Can you expand this to mention the new throwing behavior based on the constructor argument

Reply:
PTAL